test/powershell/engine/ExperimentalFeature/Get-ExperimentalFeature.Tests.ps1

```diff
@@ -1,117 +1,126 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-Describe "Get-ExperimentalFeature basic tests - Feature-Disabled" -tags "CI" {
+Describe "Get-ExperimentalFeature Tests" -tags "Feature","RequireAdminOnWindows" {
```

Comment:
The tests here will run twice after this refactoring because this test is included in `\test\tools\TestMetadata.json`.

The file `TestMetadata.json` contains the experimental features to test and the test files to run for testing that experimental feature. After the regular test run finishes, our CI script will pick up this file, and for each experimental feature listed, it will start powershell with that experimental feature enabled, and run the declared tests. This should be documented, and I will do that.

After this refactoring, tests in this file will execute in the regular test run process, and then run again when a pwsh that has `ExpTest.FeatureOne` enabled. To fix that, just change `"ExpTest.FeatureOne": [ "test/powershell/engine/ExperimentalFeature" ]` to `"ExpTest.FeatureOne": [ "test/powershell/engine/ExperimentalFeature/ExperimentalFeature.Basic.Tests.ps1" ]`